File: oop-task/7-abstraction.cpp
#include<bits/stdc++.h>
using namespace std;

class A {
private:
    int a, b, c;
 
public:
    void set(int x, int y, int z)
    {
        a = x;
        b = y;
        c = z;
    }
 
    void show()
    {
        cout << "Value of a = " << a << endl;
        cout << "Value of b = " << b << endl;
        cout << "Value of c = " << c << endl;
    }
};
 
int main()
{
    A obj;
    obj.set(10, 20, 30);
    obj.show();

    return 0;
}

File: oop-task/7-abstraction.py
class A:
    def __init__(self):
        self.__a = 0
        self.__b = 0
        self.__c = 0

    def set(self, x, y, z):
        self.__a = x
        self.__b = y
        self.__c = z

    def show(self):
        print("Value of a =", self.__a)
        print("Value of b =", self.__b)
        print("Value of c =", self.__c)

obj = A()
obj.set(10, 20, 30)
obj.show()
